.github/workflows/solutions.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         # os: [macos-latest, ubuntu-latest, windows-latest]
         os: [ubuntu-latest]
-        java-version: [16]
+        java-version: [17.0.0-ea.35]
```

Comment:
Why is this such a specific version and not just 17?

Reply:
Same comment here as the one above it.

Reply:
Made changes to add a separate step to download JDK and then use that in the setup JDK step. Doesn't use a specific version anymore and downloads the JDK from jdk.java.net.